runtime/src/bank.rs

```diff
@@ -358,13 +358,10 @@ impl Bank {
     /// Process a Transaction. This is used for unit tests and simply calls the vector Bank::process_transactions method.
     pub fn process_transaction(&self, tx: &Transaction) -> Result<()> {
         let txs = vec![tx.clone()];
-        match self.process_transactions(&txs)[0] {
-            Err(ref e) => {
-                info!("process_transaction error: {:?}", e);
-                Err((*e).clone())
-            }
-            Ok(_) => Ok(()),
-        }
+        self.process_transactions(&txs)[0].clone()?;
+        tx.signatures
+            .get(0)
+            .map_or(Ok(()), |sig| self.get_signature_status(sig).unwrap())
```

Comment:
@mvines, how do you feel about this change? This function is primarily for testing program transactions, so ignoring the ProgramError was pretty counter-intuitive. This PR fixes a handful of broken tests where others appear to be as confused as I was about the missing error.

Reply:
👍